### ISSUE TITLE
Test flake due to featured extension rotation

### ISSUE THREAD

**Issue by deboer-tim** [containers/podman-desktop#4929]
### Bug description

It seems we still have a test flake due to rotation of featured extensions:

```
verifyPodmanExtensionStatus tests/src/podman-extension-smoke.spec.ts:70:71
     69|   test('Podman extension can be disabled from Podman Extension Page', …
     70|     await openSettingsExtensionsPage();
     71|     const podmanExtensionPage = await openSettingsExtensionsPodmanPage…
       |                               ^
     72|     await podmanExtensionPage.disableButton.click();
     73|     await verifyPodmanExtensionStatus(false);
 ❯ tests/src/podman-extension-smoke.spec.ts:51:5

⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
Serialized Error: { matcherResult: { message: 'Timed out 5000ms waiting for expect(received).toBeVisible()\nCall log:\n  - locator._expect with timeout 5000ms\n  - waiting for getByLabel(\'FeaturedExtensions\').getByLabel(\'Podman\', { exact: true })\n', pass: false } }
⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/1]⎯

```

FYI @o

### Operating system

macOS 14.1

### Installation Method

None

### Version

next (development version)

**Comment by odockal** [comment]
@deboer-tim I guess that we are doing the same thing right now :) #4930
Wrong tag, btw.

**Comment by odockal** [comment]
@deboer-tim I have just merged the PR for this your exact flaky test. Try rebasing. Smoke should be passing then.

**Comment by odockal** [comment]
Should be fixed by #4908

**Comment by odockal** [comment]
Was fixed by some changes mentioned.
